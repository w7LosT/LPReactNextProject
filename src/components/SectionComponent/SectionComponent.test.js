import { screen } from "@testing-library/react";
import { renderTheme } from "../../styles/render-theme";
import { SectionComponent } from '.';

describe('<SectionComponent />', () => {
  it('should render', () => {
    const {container} = renderTheme(<SectionComponent>SectionComponent</SectionComponent>);
    expect(container).toMatchSnapshot();
  });
});
