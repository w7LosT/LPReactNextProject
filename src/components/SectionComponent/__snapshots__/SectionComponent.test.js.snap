// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionComponent /> should render 1`] = `
.c0 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 20px;
}

<div>
  <div
    class="c0"
  >
    SectionComponent
  </div>
</div>
`;
